PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+(Put an `X` inside the `[ ]` to denote check mark `[X]`.)
+
+
+- [ ] **If creating a new file :**
+  - [ ] added links to it in the README files ?
+  - [ ] included tests with it ?
+  - [ ] added description in docstrings ?
```

Comment:
I love it.
I think this line could be little more detailed.
- added description (overview of algorithm, time and space compleixty, and edge case) in docstrings?